core/store/src/trie/receipts_column_helper.rs

```diff
@@ -8,9 +8,12 @@ use near_primitives::trie_key::TrieKey;
 use near_primitives::types::ShardId;
 
 /// Read-only iterator over items stored in a TrieQueue.
-pub struct TrieQueueIterator<'a, Queue: TrieQueue> {
+struct TrieQueueIterator<'a, Q>
+where
+    Q: TrieQueue,
+{
```

Comment:
For my education, what's the difference? This one is a bit more verbose so I like the old one more but I guess it's needed? 

nit: Q -> Queue

Reply:
Ugh, this is another byproduct of me trying to do something but then reverting back... Sorry for the change, this probably wasn't required as part of this PR.

That said, there isn't too much of a functional difference between the verbose and consolidated way of writing. I personally prefer the more verbose way as it's clearer to read the constraints on the template types. Anyway, just personal preferences. If you'd like, I can revert back to original, but for now I hope it's fine if I merge in this PR?